CHANGELOG.md

```diff
@@ -82,6 +82,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Fixed query DSL `match` that supports a field name and value ([#405](https://github.com/opensearch-project/opensearch-api-specification/pull/405)) 
 - Fixed `/_mapping` with `index` in query ([#385](https://github.com/opensearch-project/opensearch-api-specification/pull/385))
 - Fixed duplicate `/_nodes/{node_id}` path ([#416](https://github.com/opensearch-project/opensearch-api-specification/pull/416))
+- Fixed Set value required in constant_keyword field type [#419](https://github.com/opensearch-project/opensearch-api-specification/pull/419)
```

Comment:
Sorry to be annoying, I promise, last one, could you please make it look like other lines? (lowercase set, quote `constant_keyword`, PR in ()). 

"Fixed required value for `constant_keyword` field type ([#419](https://github.com/opensearch-project/opensearch-api-specification/pull/419))"

Reply:
I apologize for my carelessness. I just copy, and don't know why the different. I have updated